### DISCUSSION TITLE
Bug: Replace selected text when pasting image

### DISCUSSION THREAD

**Discussion by SmartManoj** [community/community#23696] [New to GitHub]
<p>Here it works fine but not undo.<br>
How to add a tag to this post?</p>
<p>When submitting the post, if the body is empty an error box shows.<br>
When rectifying that, the alert box(the below picture) keeps shaking.</p>
<p><img src="https://user-images.githubusercontent.com/7231077/181078794-5773ed2a-6e5e-44fa-831e-f8f93583171a.png" alt="image" data-base62-sha1="2YGk8i3J2yRe4uVTYaKKJ3E2onf" width="338" height="74"><br>
What is the last line meaning?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/SmartManoj">@SmartManoj</a> :wave:</p>
<p>Thanks for sharing your concern here. It looks like that warning box will stay activated until you reply has the minimum number of characters. It should also disappear after clicking the <code>x</code> to close the box and canceling the post.</p>

**Comment by SmartManoj** [comment]
<p>Not solved<br>
Bug: Replace selected text when pasting an image in GitHub</p>
<p>I should open issues about discourse bugs in discourse forum</p>
